### PR TITLE
Day4 sequelize

### PR DESCRIPTION
**Pull request by rushikesh-thinkitive**

### CODE REVIEW

File: 6.Database/Project/CRUD.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+const mysql = require('mysql2')
+
+const pool = mysql.createPool({             // Making connection with database
+    host:'localhost',
```
Comment (imtiyazthink) [inline 6.Database/Project/CRUD.js]: Please use .env file for DB or any secrets

File: Sequelize/Project/CRUD.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+const mysql = require('mysql2')
+
+const pool = mysql.createPool({             // Making connection with database
+    host:'localhost',
```
Comment (imtiyazthink) [inline Sequelize/Project/CRUD.js:4]: Please use .env file for secrets

File: Sequelize/Project/routes/employee.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        let employee = await Employee.findByPk(empid);
+
+        // If employee not found, return 404 Not Found
+        if (!employee) {
+            return res.status(404).json({ error: 'Employee not found' });
+        }
+
+        // Update the employee's attributes
+        employee.empid = 516;
+        employee.name = "Kailash";
```
Comment (imtiyazthink) [inline Sequelize/Project/routes/employee.js:66]: Should get the details from the request instead of hardcoding them

**Review by imtiyazthink — CHANGES_REQUESTED** [review]
Code is messy 
Avoid pushing text information in .js extension, use .txt if you want
Avoid pushing logs and commented code

File: 6.Database/Project/CRUD.js (hunk 1 above)
Comment (rushikesh-thinkitive) [reply 6.Database/Project/CRUD.js]: Updated

File: Sequelize/Project/routes/employee.js (hunk 3 above)
Comment (rushikesh-thinkitive) [reply Sequelize/Project/routes/employee.js:66]: Updated

File: Sequelize/Project/CRUD.js (hunk 2 above)
Comment (rushikesh-thinkitive) [reply Sequelize/Project/CRUD.js:4]: Updated

**Review by imtiyazthink — APPROVED** [review]

**Pull request merged**